src/store/wizardSlice.ts

```diff
@@ -245,6 +245,14 @@ export const initialState: wizardState = {
   users: [],
 };
 
+const currentDateString = () => {
```

Comment:
We've got a bunch of time related helper functions in `/src/Utilities/time.ts`... you might find what you need there or if not can put this function there.

Reply:
perfect!